web/src/components/avatar/group/index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useRef } from 'react'
+import { FocusRing } from '@react-aria/focus'
+import { useButton } from '@react-aria/button'
+
+import { Icon, IconTypes } from '../../icon'
+import { Flex } from '../../layout'
+
+import { useFocusStyle } from '../../../styles/global'
+import { useAvatarStyle } from '../styles/avatar'
+import useStyles from './styles'
+
+const groupTypeIconData = {
+  private: {
+    name: 'lock',
+    width: 10,
+    height: 10,
+  },
+}
```

Comment:
Why creating a variable for this?

Reply:
Because in the future will be easy to map states by name to icon and its width/height. Now it looks a bit weird, but as we're plans to have another state except `private` I think we should keep it.